libs/contracts/src/index.ts

```diff
@@ -1,2 +1,3 @@
+export * from './lib/error-types'
```

Comment:
While updating, can we change from * to explicit exports. Better for client tree shaking and such